plugins/Kaleidoscope-LED-ActiveLayerKeys/README.md

```diff
@@ -0,0 +1,46 @@
+# LED-ActiveLayerKeys
+
+A simple way to light up the keyboard in uniform colors, depending on what layer
+one's on. Unlike [Colormap](Kaleidoscope-Colormap.md), all keys will be the same color. But
+this plugin uses considerably less resources, and is easier to set up as well. A
+perfect solution when one wants to quickly see what layer they're on, with
+minimal resources and time investment.
+
+## Using the plugin
+
+To use the plugin, one needs to include the header, and activate the effect.
+Then, one needs to configure a color map:
+
+```c++
+#include <Kaleidoscope.h>
+#include <Kaleidoscope-LEDControl.h>
+#include <Kaleidoscope-LED-ActiveLayerKeys.h>
+
+KALEIDOSCOPE_INIT_PLUGINS(LEDControl,
+                          LEDActiveLayerKeysEffect);
+
+void setup () {
```

Comment:
Could you add comment here that explains that this is an example color map for a two-layer layout that sets keys on layer 0 to $color and keys on layer 1 to $other_color?

(As it's a contrived example, it might make sense to make the example show "layer 0 is red, layer 1 is green, and layer 2 is blue" or something like that.

Reply:
I'm up for it. Should I also include an example in the PR?

Reply:
An example would be awesome. I *believe* you can just create an 'examples' directory inside the plugin and arduino will do the right thing.. I've been meaning to start moving plugin examples inside the plugins.